src/components/more-from-xl-blog/more-from-xl-blog.jsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+
+import { Link } from "gatsby";
+import PropTypes from "prop-types";
+
+import { classnames } from "../../helpers";
+
+import { Card } from "../../components/card";
+
+import * as styles from './more-from-xl-blog.module.scss';
+
+const MoreFromXlBlog = ({ data, refMoreFrom }) => {
+  const { edges } = data.allMdx;
+  return (
+    <div className={styles.moreFromXlContainer} ref={refMoreFrom} >
+      <div className={styles.titleContainer}>
+        <Link to="/" className={classnames("text__heading__three__blueTwo", styles.titleStyle)}>More From Xmartlabs Blog →</Link>
+      </div>
+      <div className={styles.blogsContainer}>
+        {edges.map(({node}) => <Card data={node} className={styles.cardStyles}/>)}
```

Comment:
Missing key here

Reply:
I don't understand what do you mean

Reply:
I meant that you should pass a key propr in the `Card` component: https://reactjs.org/docs/lists-and-keys.html#keys
Like <Card data={node} className={styls.cardStyles} key={node.id} /> (if node has id as key)